app/api/groups/get_user_batch_prefixes.feature

```diff
@@ -0,0 +1,85 @@
+Feature: List user-batch prefixes (userBatchPrefixesView)
+  Background:
+    Given the database has the following table 'groups':
+      | id | name    | type    |
+      | 13 | class   | Class   |
+      | 14 | class2  | Class   |
+      | 15 | friends | Friends |
+      | 16 | class3  | Class   |
+      | 21 | user    | User    |
+    And the database has the following table 'users':
+      | login | group_id |
+      | owner | 21       |
+    And the database has the following table 'group_managers':
+      | group_id | manager_id | can_manage  |
+      | 13       | 21         | memberships |
+      | 16       | 21         | none        |
+    And the database has the following table 'groups_ancestors':
+      | ancestor_group_id | child_group_id |
+      | 13                | 13             |
+      | 13                | 15             |
```

Comment:
This ancestor is not matching `groups_groups` as well 😔
(I still think that for feature tests, we should have a step such as "and the group ancestors match groups_groups" simply building all combination from `groups_groups` definition, to make sure they are no errors (I am sure there are many) and to make feature tests shorter/simpler)

Reply:
Fixed. For the new step, I'll think about it.